x-pack/plugins/security_solution/server/search_strategy/eql/base.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger } from 'kibana/server';
+import { ISearchStrategy } from '../../../../../../src/plugins/data/server';
+import {
+  EqlSearchStrategyRequest,
+  EqlSearchStrategyResponse,
+} from '../../../common/search_strategy/eql';
+import { EqlSearchResponse } from '../../lib/types';
+
+export const SECURITY_EQL_SEARCH_STRATEGY = 'security_eql_base';
+
+const dummyResponseFields = {
+  _shards: {
+    total: -1,
+    successful: -1,
+    failed: -1,
+    skipped: -1,
+  },
+  hits: {
+    max_score: -1,
+    total: -1,
+    hits: [],
+  },
+};
+
+export const securityEqlSearchStrategyProvider = (
+  logger: Logger
+): ISearchStrategy<EqlSearchStrategyRequest, EqlSearchStrategyResponse> => {
```

Comment:
I think this is just an oversight, but one other thing I noticed is that while `registerSearchStrategy` is generic, `getSearchStrategy` is not and so consumers of divergent search strategies would have to assert their strategy's typings, lest they get a `IEsSearchStrategy<any>`